src/restore-handlers.js

```diff
@@ -12,3 +14,19 @@ exports.mongodb = function * (filename, url) {
   yield exec('tar', ['-xf', filename])
   yield exec('mongorestore', ['-h', url.host, '--gzip'])
 }
+
+exports.postgres = function * (filename, url) {
+  const dbname = url.pathname.replace('/', '')
+  const [user, pass] = _.split(url.auth, ':')
+  yield fse.emptyDir('dump')
+
+  yield exec('tar', ['-xzf', filename])
+  try {
```

Comment:
Remove the try catch block or it won't throw exceptions to the caller function. In the current code, we're not alerted when something goes wrong. I know there is the console log, but it's better to handle all errors at the same place, thus at a higher level.